### ISSUE TITLE
Invalid command name

### ISSUE THREAD

**Issue by kohinoor12** [TomSchimansky/CustomTkinter#2274]
invalid command name "1806550247616update"
    while executing
"1806550247616update"
    ("after" script)
invalid command name "1806550247232check_dpi_scaling"
    while executing
"1806550247232check_dpi_scaling"
    ("after" script)
    
    
    
    self.win = customtkinter.CTk()
    
When generating a new window immediately after closing the previous one, the above message is generated on the console but the code works fine without any error can you please solve this as some people consider it as an error

**Comment by AshhadDevLab** [comment]
### Description:
This is caused because the window is instantiated to be looped after the initial window has been destroyed. Something like this:
```py
window.destroy()
anotherwindow.mainloop()
```

### Solution:
To avoid this you can make use of `CTkTopLevel` or create multiple `CTkFrame` widgets instead of windows and move back and fourth in them.

**Comment by AshhadDevLab** [comment]
If this answers your question mark it as solved!

**Comment by kohinoor12** [comment]
Exception ignored in: <function Variable.__del__ at 0x0000017B96258A60>
Traceback (most recent call last):
  File "AppData\Local\Programs\Python\Python39\lib\tkinter\__init__.py", line 363, in __del__
    if self._tk.getboolean(self._tk.call("info", "exists", self._name)):
RuntimeError: main thread is not in main loop


I am getting this message in the console but the application still works fine but if the message comes more then 8-10 times then the application crashes

**Comment by AshhadDevLab** [comment]
Did you try my solution?

**Comment by kohinoor12** [comment]
Yes that issue is resolved now, in my application I have a popup window that works as a preloader when different functions been called that time I'm getting the above message

**Comment by AshhadDevLab** [comment]
instead of destroying the window withdraw it first and after the other window is loaded destroy it

**Comment by AshhadDevLab** [comment]
can you somehow give a minimal reproducible code
